src-tauri/src/algorithms/stoer_wagner.rs

```diff
@@ -93,6 +118,15 @@ mod tests {
 
         let mut correct_mincut_weight_count = 0;
 
+        let graph_sw = &mut StoerWagnerGraph::new(g.clone());
+        let _mincut = stoer_wagner(graph_sw);
+
+        let nodes = vec![u, v, w, x, y, z, a, b];
+        let (s, t) = recover_mincut(graph_sw, nodes);
+
+        println!("s: {:?}", s);
+        println!("t: {:?}", t);
+
```

Comment:
Can we create a test that validates this mincut on a known graph and validates the received cuts are accurate?

Reply:
Not necessarily. At least not with the example graph because there are several global mincuts. We can't know which one the algorithm returns. I will add an issue to track this though. Might be a good idea to create a graph which has a single mincut.